CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.2.0] - TBD
```

Comment:
It looks like version 0.1.6 of stactools included stactools_planet, so 0.2.0 (not 0.1.5) feels appropriate for the next release.

Reply:
As requested in https://github.com/stac-utils/stactools/issues/214, we updated [**stactools**'s CHANGLOG](https://github.com/stac-utils/stactools/blob/08f61bdf5f51db0dec4dbbeb7472a7b15091d722/CHANGELOG.md) to better conform to https://keepachangelog.com/en/1.0.0/. As we touch any of these packages, we should probably do the same here.

v0.2.0 sounds good to me.